src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -600,58 +679,170 @@ else if ( GencodeGtfTranscriptFeature.class.isAssignableFrom(containingSubfeatur
-                                 .setProteinChange(proteinChange)
-                                 .setGcContent( sequenceComparison.getGcContent() );
+        return gencodeFuncotationBuilder.build();
+    }
 
-        // Set the Variant Classification:
-        final GencodeFuncotation.VariantClassification varClass = createVariantClassification( variant, altAllele, variantType, exon, sequenceComparison );
-        GencodeFuncotation.VariantClassification secondaryVarClass = null;
-        gencodeFuncotationBuilder.setVariantClassification( varClass );
-        if ( varClass == GencodeFuncotation.VariantClassification.SPLICE_SITE ) {
-            secondaryVarClass = getVariantClassificationForCodingRegion(variant, altAllele, variantType, sequenceComparison );
-            gencodeFuncotationBuilder.setSecondaryVariantClassification( secondaryVarClass );
-        }
+    /**
+     * Create a {@link GencodeFuncotation} for a {@code variant} that occurs in a coding region in a given {@code exon}.
+     * @param variant The {@link VariantContext} for which to create a {@link GencodeFuncotation}.
+     * @param altAllele The {@link Allele} in the given {@code variant} for which to create a {@link GencodeFuncotation}.
+     * @param gtfFeature The {@link GencodeGtfGeneFeature} in which the given {@code variant} occurs.
+     * @param reference The {@link ReferenceContext} for the current data set.
+     * @param transcript The {@link GencodeGtfTranscriptFeature} in which the given {@code variant} occurs.
+     * @param exon The {@link GencodeGtfExonFeature} in which the given {@code variant} occurs.
+     * @return A {@link GencodeFuncotation} containing information about the given {@code variant} given the corresponding {@code exon}.
+     */
+    private GencodeFuncotation createCodingRegionFuncotationForProteinCodingFeature(final VariantContext variant,
+                                                                                    final Allele altAllele,
+                                                                                    final GencodeGtfGeneFeature gtfFeature,
+                                                                                    final ReferenceContext reference,
+                                                                                    final GencodeGtfTranscriptFeature transcript,
+                                                                                    final GencodeGtfExonFeature exon) {
 
-        // Set the Coding DNA change:
-        final boolean isInCodingRegion = isVariantInCodingRegion( varClass, secondaryVarClass );
+        // Get the list of exons by their locations so we can use them to determine our location in the transcript and get
+        // the transcript code itself:
+        final List<? extends Locatable> exonPositionList = getSortedExonAndStartStopPositions(transcript);
+//        final List<? extends Locatable> exonPositionList = transcript.getExons();
```

Comment:
commented out line here, should probably be deleted

Reply:
Fixed!